### DISCUSSION TITLE
Instruments driver for Keithley 4200A-SCS Parameter Analyzer

### DISCUSSION THREAD

**Discussion by BhargavRajGuru** [microsoft/Qcodes#6402] [General]
Is there any plan to provide Keithley 4200A-SCS Parameter Analyzer drivers inside Qcodes? If it is not I would like to develop them myself. I am completely beginner here but I know python and how to use Qcodes. The driver development is the thing I am not completely aware of. If someone can help me I can try to write drivers for Keithley 4200A-SCS Parameter Analyzer.

**Comment by jenshnielsen** [comment]
I am not aware of anyone who currently has plan to develop such a driver. You can check community developed drivers in https://github.com/QCoDeS/Qcodes_contrib_drivers 

If you do start you may want to look at drivers for similar instruments from other vendors for inspiration you may also want to consult the examples here https://microsoft.github.io/Qcodes/examples/writing_drivers/index.html
